package/setup.cfg

```diff
@@ -8,3 +8,9 @@
 #keep_cythonized=True
 [wheel]
 universal = 1
+
+[build_sphinx]
+all-files = 1
+source-dir = doc/sphinx/source
+build-dir = doc/build
```

Comment:
Ok now this points to the MDA_DOCDIR variable folder. So the deploy script should still work fine